lib/features/authentication/data/datasources/signup/signup_data_source.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:dio/dio.dart';
+import 'package:jobs_tdd/features/authentication/data/models/signup/signup_user_model.dart';
+import 'package:jobs_tdd/features/authentication/domain/entities/signup/signup_data_response.dart';
+import 'package:retrofit/http.dart';
+import '../../../domain/entities/signup/signup_user.dart';
+import '../../models/signup/signup_data_response_model.dart';
+
+part 'signup_data_source.g.dart';
+
+@RestApi(baseUrl: "https://0a20-154-180-84-54.ngrok.io/api/")
```

Comment:
The `baseUrl` can change any time, Thus having it as a constant is a better solution.